ql/src/test/org/apache/hadoop/hive/ql/TestTxnCommands.java

```diff
@@ -975,7 +976,7 @@ public void testTimeOutReaper() throws Exception {
       }
     }
     Assert.assertNotNull(txnInfo);
-    Assert.assertEquals(14, txnInfo.getId());
+    Assert.assertEquals(16, txnInfo.getId());
```

Comment:
Why did we get 2 more transactions?

Reply:
because new table definition ('acidTblNestedPart') was added to TxnCommandsBaseForTests
````
    dropTables();
    setUpSchema();
````